index.html

```diff
@@ -0,0 +1,499 @@
+            <h5>CEO</h5>
+          </div>
+          <div
+            class="w-[411px] h-[404px] border-2 border-blue-200 rounded-2xl p-16 flex justify-center items-center flex-col"
+            id="solution"
+          >
+            <h3
+              class="text-[16px] text-center leading-[20px] font-medium opacity-85 mb-8"
+            >
+              Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam
+              nonumy eirmod tempor invidunt ut labore et dolore magna aliquyam
+              erat, sed diam voluptua.
+            </h3>
+            <img
+              src="./assets/profile2.svg"
+              alt="An image of a customer, a man wearing green t-shirt and smiling."
+            />
+            <h3 class="text-[20px] font-medium">Ron Wood</h3>
+            <h5>Marketing Manager</h5>
+          </div>
+          <div
+            class="w-[411px] h-[404px] border-2 border-blue-200 rounded-2xl p-16 flex justify-center items-center flex-col"
+            id="solution"
+          >
+            <h3
+              class="text-[16px] text-center leading-[20px] font-medium opacity-85 mb-8"
+            >
+              Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam
+              nonumy eirmod tempor invidunt ut labore et dolore magna aliquyam
+              erat, sed diam voluptua.
+            </h3>
+            <img
+              src="./assets/profile3.svg"
+              alt="An image of a customer, a man wearing green t-shirt and smiling."
+            />
+            <h3 class="text-[20px] font-medium">Ron Wood</h3>
+            <h5>CTO</h5>
+          </div>
+        </span>
+      </section>
+      <!-- footer -->
+      <section
+        class="relative w-[1440px] h-[700px] flex flex-col justify-center items-center bg-[url('/assets/blurGradientfooter.svg')] bg-cover bg-no-repeat bg-opacity-20 bg-bottom overflow-hidden"
+      >
+        <h1 class="text-[30px] leading-[30px] font-bold opacity-85 mb-4">
+          Are you ready to grow your business with us?
+        </h1>
+        <h4 class="text-center mb-8">
+          Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam
+          nonumy.
+        </h4>
+        <a class="bg-gray-900 rounded-md mb-72" href="#"
+          ><h6
+            class="text-white bg-black pt-4 pb-4 pr-10 pl-10 rounded-2xl w-[170px] h-[56px] font-medium"
+          >
+            View Pricing
+          </h6></a
+        >
+        <footer
+          class="w-[1440px] h-[340px] bg-gray-900 bottom-0 absolute"
+        ></footer>
+        <div class="flex justify-center pr-20 pl-32 -mt-10" id="links">
+          <div class="w-[230px] h-[212px]">
+            <h1
+              class="text-white text-[18px] leading-[30px] font-bold opacity-85 mb-4"
+            >
+              Company
+            </h1>
+            <ul
+              class="text-gray-200 text-[16px] leading-[20px] font-light opacity-85 mb-8"
+            >
```

Comment:
Avoid using arbitrary values  when tailwiind has the value by default. for. example `text-[16px]` already exist.
double check and update everywhere necessary